src/components/review/ReviewInfo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Typography from '@mui/material/Typography';
+import { CardContent } from '@mui/material';
+import ReviewRating from './ReviewRating';
+export default function ReviewInfo({ dish, restaurantName, finalScore }) {
+  return (
```

Comment:
Destructure the dish at the beginning of the component 
```suggestion
const { name, type } = dish;
  return (
```